python/python/lancedb/query.py

```diff
@@ -1112,21 +1112,21 @@ def _normalize_scores(self, results: pa.Table, column: str, invert=False):
 
     def rerank(
         self,
-        normalize="score",
         reranker: Reranker = RRFReranker(),
+        normalize: str = "score",
```

Comment:
is this a breaking change? the order changed

Reply:
yes marked